pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-pyparsing = "^2.4"
```

Comment:
```toml
python = "^3.7"
pyparsing = "^3"
```
then run `poetry update`. Otherwise `poetry.lock` still contains references to version 2, and tests run against version 2 as well.